README.md

```diff
@@ -50,152 +49,7 @@
 - Add a book.🚀
 - Borrow a book 💯
 - Add Linter👌
-  
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-The current version of node
-To have Git installed on your local machine
-Node Package Manager (npm)
-An editor as Visual Studion Code:
-
-```sh
- gem install bundler
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-```sh
- - cd oop_school_library
- - git clone git@github.com:eudondian/oop_school_library.git
- - Update your git identity by running "git config --global user.email "your email.com""
- - Update your name on git by running "git config --global user.name "your name""
-
-```
-
-
-### Install
-
-Install this project with:
-
-To Install this, 
-
-```sh
- Run bundle install
-
-```
-
-### Usage
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-
-  ```sh
-  Run "rubocop --fix to fix linters"
-
-```
-or
-
-```sh
-  Run "rubocop -A fix to fix linters"
-
-```
-
-### Deployment
-
-You can deploy this project using:
-
-github pages
-
-```sh
-npm run deploy
-
-```
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-
-👤 **Author1**
-
-- GitHub: [@Click here](https://github.com/eudondian)
-- Twitter: [@Click here](https://twitter.com/EUdondian)
-- LinkedIn: [@Click here](https://www.linkedin.com/in/esther-udondian/)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
- - [Contact form page]
- - [A feature for a more detailed demographic data]
- - [A feature for updating the list of countries]
-
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](https://github.com/eudondian/opp-school-library/issues).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
-
-## ⭐ Show your support <a name="support"></a>
-
-If you like this project you can leave a star to it.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-- I would like to thank Microverse for giving us an opportunity to learn more about software Development.
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
```

Comment:
- [ ] The previous reviewer requested that you remove only the section description on line 64, but you removed almost all the readme. Kindly bring back these deleted parts for a good documentation.